### DISCUSSION TITLE
How to debug `DataCloneError` problem?

### DISCUSSION THREAD

**Discussion by alexgorbatchev** [oven-sh/bun#13065] [General]
I have bundled my app with bun, however when I try to run it i get `DataCloneError: The object can not be cloned.` The error goes away if i copy over `node_modules`. There's no stack trace or anything else. I've searched the docs and all flags, can't find anything that could help me debug this issue. Is there a flag or env variable I can set for more verbose output?

**Comment by alexgorbatchev** [comment]
Figured it out

```shell
 strace -e trace=openat -f bun ./index.js
```

**Comment by moda20** [comment]
@alexgorbatchev what was the cause of your issue ? i am having the same. but it doesn't occur all the time

**Comment by alexgorbatchev** [comment]
To be honest, I don't remember now, but it was pretty obvious from strace.
